Basic_CNN_Architecture

```diff
@@ -1,19 +1,45 @@
-import torch
-import torch.nn as nn
```

Comment:
Did you remove the import statements by accident?

Reply:
Yeah, my bad. Was trying things out in jupyter and had the imports in another cell. Will add it back after I work on it some more.